doc/index.rst

```diff
@@ -332,10 +332,16 @@ Use .rosinstall file to specify the depended packages source repository
 
 WARNING: In all cases where you want to utilize `.rosinstall` (or similar name) files, be sure to set `USE_DEB` as `false`, or simply not define it.
 
-For using a rosinstall file located locally within the repository:
+For using a rosinstall file located locally within the repository, define both two variables as:
```

Comment:
You don't have to define both if you stick to the default of `ROSINSTALL_FILENAME`